src/NumberTheory/vinberg.jl

```diff
@@ -0,0 +1,295 @@
+###################################################################
+# Vinberg's algorithm
+###################################################################
+
+@doc raw"""
+    _check_v0(Q::ZZMatrix, v0::ZZMatrix) -> ZZMatrix
+
+Checks if the inner product of the given v0 is positive or generates such a v0 if none is given.
```

Comment:
```suggestion
Checks if the inner product of the given `v0` is positive or generates such a `v0` if none is given.
```
Similar in all other docstrings.